codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/customizations/FallibleOperationsDecorator.kt

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+package software.amazon.smithy.rust.codegen.server.smithy.customizations
+
+import software.amazon.smithy.model.Model
+import software.amazon.smithy.model.shapes.MemberShape
+import software.amazon.smithy.model.shapes.OperationShape
+import software.amazon.smithy.model.shapes.ServiceShape
+import software.amazon.smithy.model.shapes.StructureShape
+import software.amazon.smithy.model.traits.ErrorTrait
+import software.amazon.smithy.model.traits.RequiredTrait
+import software.amazon.smithy.model.transform.ModelTransformer
+import software.amazon.smithy.rust.codegen.smithy.customize.RustCodegenDecorator
+
+/**
+ * Add at least one error to all operations in the model.
+ *
+ * When this decorator is applied, operations that do not have a Smithy error attatched,
+ * will return `Result<OperationOutput, InternalServerError>`.
+ *
+ * To enable this decorator, create a file called `META-INF/services/software.amazon.smithy.rust.codegen.smithy.customize.RustCodegenDecorator`
+ * containing `codegen-server/src/main/resources/META-INF/services/software.amazon.smithy.rust.codegen.smithy.customize.RustCodegenDecorator`.
```

Comment:
@crisidev shouldn't this change to match the updated decorator class name - FallibleOperationsDecorator?

I am trying to get this to work from my separate gradle build root, and I see the jar containing the meta-inf file, but the decorator is not running (I added a print to make sure). I am probably missing something regarding the gradle build structure but hard to figure out what. Any chance you could show me how this can be applied to the codegen-server-test project perhaps? Sorry for the hassle, but I'm a bit lost after trying a bunch of things. Thanks

Reply:
Damn, this is completely wrong.. it should contain the java path of the decorator you want to use: `software.amazon.smithy.rust.codegen.server.smithy.customizations.FaillibleOperationsDecorator`.